app/views/documents/index/_results.html.erb

```diff
@@ -38,6 +38,10 @@
           <%= t "documents.index.search_results.state",
             state: t("user_facing_states.#{document.user_facing_state}.name")
           %>
+          <br/>
+          <%= t "documents.index.search_results.last_edited_by",
+            user: document.last_editor&.name || document.creator&.name || I18n.t("documents.index.search_results.unknown_creator")
+          %>
```

Comment:
Should we change this key to be more generic such as unknown user?

Reply:
I think this is reasonable given we're using it in a couple of places now - changed here: https://github.com/alphagov/content-publisher/pull/336/commits/b21fd9edfe5dbfd6cf18e0ba87ad4d97a601d63d